docs/dashboards/entity-dashboard.asciidoc

```diff
@@ -0,0 +1,57 @@
+[[detection-entity-dashboard]]
+= Entity dashboard
+
+The Entity Analytics dashboard provides a centralized view of critical entity metrics, which include hosts, users, and anomalies. Use this information to determine which emerging threats pose a higher risk to your environment so you can respond quickly. 
+
+
+.Requirements
+[sidebar]
+--
+The following is required to display the Entity Analytics dashboard: 
+
+* A https://www.elastic.co/pricing/[Platinum subscription] or higher.
+* To display host and user risk scores, the host risk score and user risk score features must be enabled. For more information, refer to Enable host risk score and <<deploy-user-risk-score, Enable user risk score>>.
+* To display notable anomalies, the following machine learning jobs need to be installed and run. You can do this directly from the dashboard's Notable anomalies table: 
```

Comment:
> ... the following machine learning jobs need to be installed and run. You can do this directly from the dashboard's Notable anomalies table:

On 8.5 users can't install jobs from the table. It will link them to the ml/jobs page or to the documentation.